exporter/coralogixexporter/README.md

```diff
@@ -96,12 +103,7 @@ When using OpenTelemetry Collector with [k8sattribute](https://github.com/open-t
 exporters:
   coralogix:
     # The Coralogix traces ingress endpoint
-    traces:
-      endpoint: "ingress.coralogix.com:443"
-    metrics:
-      endpoint: "ingress.coralogix.com:443"
-    logs:
-      endpoint: "ingress.coralogix.com:443"
+    endpoint: "ingress.coralogix.com:443"
```

Comment:
Same as above we should support global domain and domain per endpoint by default you'll use the global but you can override telemetry if needed. Also we should be able to enable/disable telemetry